M7/7.ICE.1/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import 'regenerator-runtime/runtime' // for using async/await
+
+function App () {
```

Comment:
Bro we shouldnt be using a component here, lets stick to the flow step by step! Check out the preclass to see how its done! can follow the setup in 7.1